packages/block-library/src/column/block.json

```
@@ -12,6 +12,9 @@
 		"width": {
 			"type": "string"
 		},
+		"allowedBlocks": {
+			"type": "array"
+    },
```

Indentation is off here
```suggestion
		},
```